### ISSUE TITLE
Refine ortholog/paralog/whatalog designations

### ISSUE THREAD

**Issue by sammyjava** [LegumeFederation/intermine-bio-sources-legfed#3]
GeneFamilyProcessor and HomologueProcessor make assumptions about ortholog vs paralog that should be improved. We don't really know which genes are orthologous simply from gene family assignments, presumably only one (or two given duplication) is orthologous to a gene from another species (thanks, Chris). So we probably need to go with homologue for inter-species. Technically, I think paralogues are results of gene duplication, so even genes within the same species and gene family are not necessarily paralogues, technically. **So maybe leave the "type" field blank since we can't really distinguish orthologs and paralogs just from gene families?**

**Comment by adf-ncgr** [comment]
I think the larger issue to be addressed here is probably what should be included in the set of "friend-making" genes (amalogues?). Our current practice of including everything in the gene family dilutes friendship to the point of questionable utility for the user, IMHO. For genes that have been included in trees, we can probably implement a strategy based on the tree topology. For genes assigned to families after the last tree-build, I'm not sure what we would do. worth further discussion, perhaps among the @LegumeFederation/gene-families team (with you as our honored guest) and/or the @LegumeFederation/federators team.

**Comment by sammyjava** [comment]
Indeed. For the record, FlyMine combines Treefam and Panther. MedicMine uses "Gene families representing clade-specific orthology/paralogy relationships via PhytoMine web services" (real-time request which currently looks broken, at least on the example I tried) which I think is broader than what we have if my look at Phytozome is representative of what Vivek grabs.
